docs/content/latest/api/ysql/exprs/func_currval.md

```diff
@@ -4,7 +4,7 @@ headerTitle: currval()
 linkTitle: currval()
 description: Returns the last value returned by the nextval() function for the specified sequence in the current session.
```

Comment:
This description doesn't agree with the page title
```suggestion
description: Returns the last value returned by the currval() function for the specified sequence in the current session.
```

Reply:
@polarweasel I think this is actually correct just ambiguously written. `nextval` advances the sequence and returns the new value. `currval` just returns the current value. So it ends up returning the value of the latest `nextval` call.
Postgres docs: https://www.postgresql.org/docs/11/functions-sequence.html phrase it as:
```
Return value most recently obtained with nextval for specified sequence
```
cc @tanujnay112